save bridgeID

dgomes opened the pull request:
keep bridgeID inside configuration.yaml

dgomes committed:
Merge remote-tracking branch 'oskarn97/master'

dgomes committed:
save bridgeID

oskarn97 commented:
Thanks!
BridgeID is "bridge" by default and not the iEEE Adress because this is used in the FHEM plugins...
I know a unique ID would maybe be a better idea but the module would not work anylonger. Maybe we can make this optional?

dgomes committed:
default values

Don’t change the configured bridgeID (backward compatible with bridgeID
= “bridge”)

dgomes commented:
As a compromise:
- default configuration.yaml keeps "bridge"
- removing the default parameter from the config file will auto save the ieee address

oskarn97 commented:
cool idea, thanks!
